wikipron/cli.py

```diff
@@ -71,6 +71,11 @@ def _get_cli_args(args: List[str]) -> argparse.Namespace:
             "overridden. If not given, results appear in stdout."
         ),
     )
+    parser.add_argument(
+        "--segment",
+        action="store_true",
+        help="Separate pron segments with unambiguously with added whitespace."
```

Comment:
Should we make this true by default? Any thoughts?

Reply:
I did consider it, but then it wouldn't really work to name the flag and variables `--segment` anymore, and I guess I grew attached to that. Do you have any suggestions?